rt/src/runtime.rs

```diff
@@ -22,13 +22,17 @@ use crate::process::{NativeAsyncMethod, Process};
 use crate::scheduler::{number_of_cores, pin_thread_to_core};
 use crate::stack::Stack;
 use crate::state::{MethodCounts, RcState, State};
+
 use std::env::args_os;
 use std::io::{stdout, Write as _};
 use std::process::exit as rust_exit;
 use std::thread;
 
 #[cfg(unix)]
-fn ignore_sigpipe() {
+pub type SigHandlerT = usize;
+pub const SIGPIPE: i32 = 13;
+pub const SIG_IGN: SigHandlerT = 1 as SigHandlerT;
+extern "C" {
```

Comment:
```suggestion
const SIGPIPE: i32 = 13;
const SIG_IGN: usize = 1;

extern "C" {
```

The `pub` isn't necessary, and the `SigHandlerT` type is redundant as we can just use `usize` instead.